homeassistant/components/axion_dmx/config_flow.py

```diff
@@ -0,0 +1,82 @@
+"""Config flow for Axion Lighting integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from libaxion_dmx import AxionDmxApi
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import selector
+
+from .const import _LOGGER, CONF_CHANNEL, CONF_LIGHT_TYPE, DOMAIN
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_CHANNEL): vol.All(vol.Coerce(int), vol.Range(min=1)),
+        vol.Required(CONF_LIGHT_TYPE): selector.SelectSelector(
+            selector.SelectSelectorConfig(
+                options=[
+                    selector.SelectOptionDict(value="White", label="White"),
+                    selector.SelectOptionDict(
+                        value="Tunable White", label="Tunable White"
+                    ),
+                    selector.SelectOptionDict(value="RGB", label="RGB"),
+                    selector.SelectOptionDict(value="RGBW", label="RGBW"),
+                    selector.SelectOptionDict(value="RGBWW", label="RGBWW"),
```

Comment:
Why are they still uppercase? I would expect them to be `snake_case`

Reply:
Addressed this!